src/ci/docker/host-x86_64/dist-x86_64-linux/Dockerfile

```diff
@@ -55,6 +55,7 @@ RUN ./build-clang.sh
 ENV CC=clang CXX=clang++
 
 # rustc-perf version from 2023-03-15
```

Comment:
oli forgot to bump this so let's do it here so that everything is correct whenever we ref to that commit, here and in the opt-dist windows module.

```suggestion
# rustc-perf version from 2023-05-30
```

Reply:
Thanks!